README.md

```diff
@@ -445,6 +445,7 @@ Variable | Description | Default value
 `JIGASI_TRANSCRIBER_ADVERTISE_URL` | Jigasi will post an URL to the chat with transcription file | true
 `DISABLE_HTTPS` | Handle TLS connections outside of this setup | 1
 `ENABLE_HTTP_REDIRECT` | Redirect HTTP traffic to HTTPS (necessary for Let's Encrypt) | 1
+`ENABLE_WEBSOCKET` | Enabled xmpp traffic over websockets | 0
```

Comment:
Leet's call this ENABLE_XMPP_WEBSOCKET, since JVB datachannels can also be configured to use WS.

Reply:
Done